projects/js-packages/components/components/boost-score-graph/use-boost-score-transform.ts

```diff
@@ -28,6 +28,6 @@ export function useBoostScoreTransform(
 			);
 		}
 
-		return [ timestamps, ...valueArray ];
+		return [ timestamps, ...valueArray ] as ScoreGraphAlignedData;
```

Comment:
timestamps and value array should be defined as `number[]` and then this could be inferred without `as`

Reply:
The `valueArray` would always have two child arrays because of `keysToExtract`. Which typsecript wasn't able to detect and that's why I used `as`. Anyway, I have refactored it to explicitly add each array element.